Common/src/main/java/com/blamejared/crafttweaker/api/zencode/IPreprocessor.java

```diff
@@ -44,6 +46,13 @@ record Match(IPreprocessor preprocessor, int line, String content) {}
      */
     Pattern PREPROCESSOR_PATTERN = Pattern.compile("#\\w+ ?");
     
+    /**
+     * Logger that can be used within a preprocessor to log messages.
+     *
+     * @since 10.1.0
+     */
+    Logger PREPROCESSOR_LOGGER = CommonLoggers.zenCode();
```

Comment:
should this not be in CommonLoggers?

Reply:
The idea was to provide an alternative to accessing the field without going through `CommonLoggers`, ensuring that people could use the field reference and not worry about what the actual subsystem name is. This would allow us to potentially change it in the future to different or more specific subsystems if the need arises.

Would you prefer having `CommonLoggers.preprocessor()` regardless?